README.md

```diff
@@ -99,15 +133,15 @@ Select the InfluxDB target you wish to publish the data to.
 
 From the "Advanced" tab you can choose to set a custom prefix for your `project_name` field,
 a custom project name to be used instead of the default job name and custom fields and tags
-for your `jenkins_data` metric.
+for your all your metrics.
```

Comment:
Slight nitpicking, but only tags are for all metrics, custom fields are still only for `jenkins_data`.

Reply:
Thanks!